DevDeployment/check_service_status_webjob.ps1

```diff
@@ -0,0 +1,49 @@
+Param(
```

Comment:
Why is this file in DevDeployment folder, whereas the QALiveDeployment is in the templates subfolder? Should these be consistent?

Also do we need this duplication, can't there be a common folder referenced?

Reply:
Hi Jason,
Since we publish the deployment folder as an artifact during the build stage and redownload it during the deploy stages there would need to be some changes to the way the pipeline handles this in order to share a common folder across the pipelines. At the moment we have a separate dev folder since we deploy the mock API during this stage alone, and this requires a unique set of terraform files and associated scripts. The status check duplicate can be handled but since it may require some deeper re-thinking of the pipeline operation and I suggest we return to this as a future tech debt item.

Reply:
Thanks